packages/gatsby-theme-bulmaio/src/common/SiteConfig.ts

```diff
@@ -5,7 +5,7 @@ to get it.
 
  */
 
-import { NavbarEndProps } from '../components/navbar/NavbarEnd';
+import { NavbarEndProps } from 'gatsby-theme-bulmaio/src/components/navbar/NavbarEnd';
```

Comment:
Is there a reason for converting these from relative to absolute?

Reply:
Fixed in the latest commit.